clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -1,35 +1,42 @@
 import { h, FunctionComponent } from "preact";
 import { useState, useEffect } from "preact/hooks";
-import { ButtonType } from "../lib/consent-types";
-import ConsentBannerButton from "./ConsentBannerButton";
+import {
+  ButtonType,
+  ExperienceConfig,
+  ExperienceComponent,
+} from "../lib/consent-types";
+import Button from "./Button";
 import "../lib/banner.module.css";
 import { useHasMounted } from "../lib/hooks";
 
 interface BannerProps {
+  experience: ExperienceConfig;
+  privacyCenterUrl: string;
   onAcceptAll: () => void;
   onRejectAll: () => void;
-  privacyCenterUrl: string;
-  bannerDescription?: string;
```

Comment:
instead of taking these individually, I thought it made sense to just pass in the whole `experience` obj